src/test/java/com/hocs/test/pages/decs/Search.java

```diff
@@ -59,73 +60,145 @@ public void enterSearchCriteria(String criteria, String value) {
         String validatedHeaderText;
         switch (criteria.toUpperCase()) {
             case "CASE TYPE":
+                List<WebElementFacade> listOfCaseTypeCheckboxes = findAll("//fieldset[@id='caseTypes']//input[not(@checked)"
+                        + "]/following-sibling::label");
```

Comment:
why define this list outside of the relevant switch case? It wont be needed unless Random is the value.